src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -237,20 +258,12 @@ def get(self, request: Request, project: Project) -> Response:
                     break
 
             # Post-processing before exiting scope context
-            found: bool = result["sourceUrl"] is not None
-            scope.set_tag("stacktrace_link.found", found)
-            scope.set_tag("stacktrace_link.auto_derived", derived)
             if current_config:
                 result["config"] = current_config["config"]
-                if not found:
-                    result["error"] = current_config["outcome"]["error"]
-                    # When no code mapping have been matched we have not attempted a URL
-                    if current_config["outcome"].get("attemptedUrl"):
-                        result["attemptedUrl"] = current_config["outcome"]["attemptedUrl"]
-                    if result["error"] == "stack_root_mismatch":
-                        scope.set_tag("stacktrace_link.error", "stack_root_mismatch")
-                    else:
-                        scope.set_tag("stacktrace_link.error", "file_not_found")
+                try:
```

Comment:
If by any chance tagging failed to work we should not prevent the endpoint to fail for the UI.